### DISCUSSION TITLE
Unable to configure wallet for both Mobile and Injected

### DISCUSSION THREAD

**Discussion by punithbm** [WalletConnect/walletconnect-monorepo#2468] [Developer Support] [answered]
Our wallet is available for both mobile as app and browser as extension.  However after creating and publishing the project in the wallet connect, In web3modal only mobile QR code shows up and the extension installed in the browser doesnt get detected. 
This is the configuration i have under walletconnect 
-------------
<img width="897" alt="Screenshot 2023-05-25 at 7 28 45 AM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/13044958/8ffd60ad-5070-48a0-af1f-5aba86cb3947">
-------------
<img width="984" alt="Screenshot 2023-05-25 at 7 29 09 AM" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/13044958/93563d52-e102-49cc-86f4-59cbb87ecddd">
-------------

Can you please help us out with the configuration to be updated in order make the wallet available for both injected and mobile

**Comment by finessevanes** [comment]
Hey - I let our team know. You should be getting a response soon.

**Comment by Cali93** [comment, marked as answer]
Hi @punithbm,
Can you remove the `window.` part ?
It should resolve the issue once the submission is approved.

**Reply by punithbm** [reply]
Ah ok got it. Thanks
